### PR TITLE
[#132] convert json based API to HTML returning api

### PR DESCRIPTION
**Pull request by shailee-m**
## Fixes
Related to https://github.com/creativecommons/project_creativecommons.org/issues/132 by @brylie 

## Description
Replaced API functions that return JSON with HTML content generation logic. Currently, using the walker class to generate the HTML. Can also use twig here if custom HTML output is required.
CC @brylie  : Added summary in [this comment](https://github.com/creativecommons/project_creativecommons.org/issues/132#issuecomment-979363126)

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

### CODE REVIEW

File: inc/filters.php [hunk 1]
```diff
@@ -1,5 +1,15 @@
 <?php
 class CC_Org_Filters {
+  public static function add_html_content()
+  {
+    $style_imports = array(
+      "@import url(https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css);",
+      //  TODO: Replace with appropriate creative commons url.
+      "@import url(http://127.0.0.1:8000/wp-content/themes/creativecommons-base/assets/css/styles.css);"
```
Comment (brylie) [inline inc/filters.php:8]: We can get the site URL via WordPress [`get_site_url`](https://developer.wordpress.org/reference/functions/get_site_url/)

**Review by brylie — CHANGES_REQUESTED** [review]

**Comment by brylie** [comment]
For what it's worth, we might not want to remove the JSON endpoints. The original task is just to add HTML endpoints.

**Comment by shailee-m** [comment]
Closing this PR as we are looking into vue components

**Comment by brylie** [comment]
@shailee-m, would you like to create a Vue component that fetches and renders the WordPress navigation menu?

**Pull request closed without merging**